rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -28,8 +31,13 @@ void opcode_0001(const ScriptArguments& args, const ScriptInt time) {
 */
 void opcode_0002(const ScriptArguments& args, const ScriptLabel arg1) {
 	RW_UNIMPLEMENTED_OPCODE(0x0002);
-	RW_UNUSED(arg1);
-	RW_UNUSED(args);
+	auto thread = args.getThread();
+	if (arg1 < 0) {
+		thread->programCounter = thread->baseAddress - arg1;
+	}
+	else {
+		thread->programCounter = arg1;
+	}
```

Comment:
Maybe beyond the scope: This is formatted differently in all other cases (ternary bool operator). Lines 972, 985, 1030 etc.